sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -152,9 +153,6 @@ def _do_send_invoice(self, invoice, domain_filter):
         invoice_send_wizard._compute_composition_mode()
         invoice_send_wizard.onchange_template_id()
         invoice_send_wizard.onchange_is_email()
-        _logger.info(
```

Comment:
This was just cleanup from previous commit, while I was figuring out why tests failed.